api/apps/api/src/modules/authentication/password-recovery/password-recovery.controller.ts

```diff
@@ -0,0 +1,76 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Body,
+  Controller,
+  Post,
+  Headers,
+  UnauthorizedException,
+  Patch,
+} from '@nestjs/common';
+import {
+  ApiHeader,
+  ApiNoContentResponse,
+  ApiOperation,
+  ApiProperty,
+  ApiTags,
+} from '@nestjs/swagger';
+import { IsEmail, IsString, MaxLength } from 'class-validator';
+import { PasswordRecoveryService } from './password-recovery.service';
+
+class PasswordRecoveryRequestDto {
+  @IsEmail()
+  @ApiProperty()
+  email!: string;
+}
+
+class SetPasswordDto {
+  @ApiProperty()
+  @IsString()
+  @MaxLength(72)
+  passwordConfirm!: string;
+}
+
+@Controller(`${apiGlobalPrefixes.v1}/users/me`)
```

Comment:
Yes, because we're operating on a user entity, and here in the system, changes on the user entity are made with `me` pseudo id.
Also, app/docs/roadmap.md described those endpoints.